modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/slot/SlotManagerClusterMode.java

```diff
@@ -430,7 +432,7 @@ public void reAssignSlotWhenNoSubscribers(String nodeId, String queueName) throw
 		}
 	}
 
-	protected Long getLastPublishedIDByNode(String nodeID) throws AndesException {
+	protected Long getLocalSafeZone(String nodeID) throws AndesException {
```

Comment:
nope. this is a naming confusion since the old method is re-used. will refactor.